.eslintrc.js

```diff
@@ -19,7 +19,6 @@ module.exports = {
       '@typescript-eslint/interface-name-prefix': 'off',
       '@typescript-eslint/explicit-function-return-type': 'warn',
       '@typescript-eslint/explicit-module-boundary-types': 'off',
-      '@typescript-eslint/no-explicit-any': 'error',
```

Comment:
Je kon hier ook `error `veranderd hebben naar `warn` als we willen weten wanneer iets `any` is. Als `any` niet veel meer zal voorkomen, kan je ook boven de codelijn waar het type in wordt gebruikt volgende plaatsen: 	
`// eslint-disable-next-line @typescript-eslint/no-explicit-any`
Dit vertelt dat de codelijn onder deze commentaarregel een uitzondering is op de regel en zal dus ook worden genegeerd door de regel die controleert of het type `any `wordt gebruikt.